cmd/kubeadm/app/master/apiclient.go

```diff
@@ -129,6 +102,36 @@ func standardLabels(n string) map[string]string {
 	}
 }
 
+func WaitForAPI(client *clientset.Clientset) {
+	start := time.Now()
+	wait.PollInfinite(apiCallRetryInterval, func() (bool, error) {
+		// TODO: use /healthz API instead of this
+		cs, err := client.ComponentStatuses().List(v1.ListOptions{})
+		if err != nil {
+			if apierrs.IsForbidden(err) {
+				fmt.Print("\r[apiclient] Waiting for the API server to create RBAC policies")
```

Comment:
please `fmt.Println("[apiclient] Waiting for API server authorization")` as in https://github.com/kubernetes/kubernetes/pull/40154